ios/Classes/MapboxMapFactory.swift

```diff
@@ -76,41 +76,41 @@ class MapboxMapFactory: NSObject, FlutterPlatformViewFactory {
         return mapOptions
     }
 
-    private func createCameraOptions(args: [String: Any]) -> CameraOptions {
-        var cameraOptions = CameraOptions()
-        if let cameraOptionsMap = args["cameraOptions"] as? [Any] {
-            var center: CLLocationCoordinate2D?
-            var padding: UIEdgeInsets?
-            var anchor: CGPoint?
-            let zoom: CGFloat? =  cameraOptionsMap[3] as? CGFloat
-            let bearing: CLLocationDirection? =  cameraOptionsMap[4] as? CLLocationDirection
-            let pitch: CGFloat? =  cameraOptionsMap[5] as? CGFloat
-
-            if let centerMap = cameraOptionsMap[0] as? [String: Any] {
-                center = convertDictionaryToCLLocationCoordinate2D(dict: centerMap)
-            }
+    private func createCameraOptions(args: [String: Any]) -> CameraOptions? {
+        guard let cameraOptionsMap = args["cameraOptions"] as? [Any] else {
+            return nil
+        }
```

Comment:
Git shows a bunch of changes, but the only thing I did was to make the return type optional and add early return in case camera options were not provided.
(This is needed because even empty camera options `CamerOptions()` seem to override style default camera).